src/components/ShareBar/ShareBar.js

```diff
@@ -101,14 +100,7 @@ class ShareBar extends React.Component<Props, State> {
 
     if (!this.state.isExpanded) return expandButton;
 
-    // The share button lib we're using is using window.open() by default which
-    // seems to not work in our Cordova app for some reason. If we are a Cordova
-    // app then we configure the share buttons to use window.location.href which
-    // works in Cordova.
-    // If we are on web we just use the default behavior of the share buttons.
-    const linkOpeningViaLocationHrefProps = isCordova()
```

Comment:
Do the share buttons still work correctly? If not, should we switch them off and replace them with native the sharing APIs?

Reply:
On iOS everything but the e-mail share link work fine, will look into it.